container_linux.go

```diff
@@ -125,7 +126,17 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 		Path: c.initPath,
 		Args: c.initArgs,
 	}
-	cmd.Stdin = p.Stdin
+	if p.Stdin != nil {
+		stdin, err := cmd.StdinPipe()
+		if err != nil {
+			return nil, err
+		}
+		go func() {
+			io.Copy(stdin, p.Stdin)
+			stdin.Close()
+		}()
+	}
+
```

Comment:
This should be the same as `cmd.Stdin = p.Stdin` except that in the original statement, `exec.Cmd` is doing it internally ( creating a os.Pipe and copied from cmd.Stdin ), so i'm not sure why the original didnt work. Do you have a test case ?